packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -73,33 +94,23 @@ const useRootStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusSmall),
   },
   circular: {
-    ...shorthands.borderRadius(tokens.borderRadiusCircular),
+    // Set by useRootClassName
   },
 
   // border (all appearances except ghost)
```

Comment:
This comment no longer applies, it should be changed to something like

`// Do not display the border pseudo-element when appearance is ghost.`